tests/EventExport/Command/ExportEventsJSONDeserializerTest.php

```diff
@@ -1,152 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace CultuurNet\UDB3\EventExport\Command;
-
-use CultuurNet\UDB3\Deserializer\DeserializerInterface;
-use CultuurNet\UDB3\Deserializer\MissingValueException;
-use CultuurNet\UDB3\EventExport\EventExportQuery;
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\TestCase;
-use ValueObjects\StringLiteral\StringLiteral;
-use ValueObjects\Web\EmailAddress;
-
-class ExportEventsJSONDeserializerTest extends TestCase
```

Comment:
Isn't this still needed?

Reply:
Wow, did I remove the wrong file?

Reply:
Ah it referenced the CSV one I see now in d567ce4 so it's an understandable mistake :)